README.md

```diff
@@ -48,11 +47,8 @@ There are two builder images under ./pipeline folder:
 
 
 Note on CI:
-On each merge under a workflow directory a matching github workflow executes
-an image build, generating manifests and a PR create on the [helm chart repo][3].
-The credentials of this repo are an org level secret, and the content is from a token
-on the helm repo with an expiry period of 60 days. Currently only the repo owner (rgolangh) can
-recreate the token. This should be revised.
+On each PR merge under a workflow directory github workflow executes an image build that generates manifests and a new PR is created in the [helm chart repo][3].
```

Comment:
nit: s/github/GitHub/
nit: s/helm chart/Helm Chart/ 
(Possibly for all the occurrences :) ) 

What about rephrasing the sentence for better clarity? 
E.g. "For every PR merged in the workflow directory, a GitHub Action runs an image build to generate manifests, and a new PR is automatically generated in the Helm Chart repository"